frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -752,10 +752,10 @@ function ReaderHighlight:onHold(arg, ges)
             table.insert(boxes, self.selected_word.sbox)
             self.view.highlight.temp[self.hold_pos.page] = boxes
         end
-        UIManager:setDirty(self.dialog, "ui")
-        --- @todo only mark word?
         -- Unfortunately, CREngine does not return good coordinates
-        -- UIManager:setDirty(self.dialog, "partial", self.selected_word.sbox)
+        -- UIManager:setDirty(self.dialog, "ui")
+        -- But now it does:
+        UIManager:setDirty(self.dialog, "partial", self.selected_word.sbox)
```

Comment:
Using `partial` here means it'll make the "pages until flash" counter tick. Not sure if we want that here?

Reply:
Made it "ui".
First time testing this "ui" on my GloHD, I had it doing a faster animation white>black>grey>black>gray instead of the usual white>black>grey.
After a koreader restart, it was fine...
My GloHD is getting more and more inconsistent...

Reply:
Turned out this is temperature dependant :/
At 21°, I get this extra animation, white>black>grey>black>gray 
At 24°, I only get the white>black>grey ...

Reply:
24°? I mean, it's not super cold, but… :-P

![Screenshot_2020-12-31_13-29-20](https://user-images.githubusercontent.com/202757/103410572-37398600-4b6c-11eb-8925-bd2ddde80456.png)

Reply:
That's the device temperature, from `/sys/devices/platform/imx_epdc_fb/graphics/fb0/temperature` on my Kobo.
In my hands after 20mn of reading, it can reach 30° :)

Reply:
Yep, colder means slower, and hotter means faster refreshes (and possibly slightly different waveform table). Until it gets too hot and it's slow as all hell again ^^.